src/xdocs/config_naming.xml

```diff
@@ -911,8 +929,8 @@ class MyClass {
         </p>
         <source>
 &lt;module name=&quot;ConstantName&quot;&gt;
-  &lt;property name="applyToPublic" value="false"/&gt;
-  &lt;property name="applyToProtected" value="false"/&gt;
+&lt;property name="applyToPublic" value="false"/&gt;
+&lt;property name="applyToProtected" value="false"/&gt;
```

Comment:
revert this, you should not change this indentation

Reply:
It was showing an error that's why I changed.

Reply:
there should be no error, please revert and share error message